services/types.go

```diff
@@ -1,6 +1,7 @@
 package services
 
 type OnboardingTicket struct {
-	ID             string `json:"id"`
-	ExpirationDate int64  `json:"expirationDate,string"`
+	ID                string `json:"id"`
+	ExpirationDate    int64  `json:"expirationDate,string"`
+	StorageQuotaInGiB *int   `json:"storageQuotaInGiB,omitempty"`
```

Comment:
There is no need to have this as a pointer. As of the "omitempty" tag a value of 0 will not be marshaled and 0 has no meaningful value as a quota